tools/pkg-config/csfml-graphics.pc.in

```diff
@@ -0,0 +1,13 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/include
+
+Name: CSFML-graphics
+Description: The Simple and Fast Multimedia Library for C, graphics module.
+URL: http://www.sfml-dev.org
+Version: @PROJECT_VERSION@
+Requires: csfml-window
```

Comment:
I assume pkgconfig will automatically know the transitive dependency, correct?

Technically speaking, csfml-graphics also depends on csfml-system, not sure if that should be reflected, when csfml-window already depends on csfml-system